ts/quill/signal-clipboard/index.ts

```diff
@@ -67,7 +67,7 @@ export class SignalClipboard {
 
     const { ops } = contents;
 
-    if (ops === undefined) {
```

Comment:
how about making this a falsy check instead?

Reply:
ah yes you're right `ops` is an Array of `DeltaOperation`. In that case I propose we change it to `if (!ops || !ops.length)` just to be a little more defensive. I think the API only returns undefined or Array but just in case.

Reply:
Sounds good! Should be updated now.